### ISSUE TITLE
AvatarGroup:  aria-label for avatar group is not descriptive

### ISSUE THREAD

**Issue by summer2200** [SAP/ui5-webcomponents#10998]
### Describe the bug

When there's no hidden items in an avatar group, when focus on it, the screen reader announces 'Conjoined Avatar, 1 displayed, 0 hidden,' which is not meaningful or descriptive. 
There are no avatars/employees hidden so this label text ' 0 hidden 'does not add any useful information for non-sighted users.

And the component does not have an attribute like "accessibleName" which let users to set the aria-label by themselves.

<img width="1170" alt="Image" src="https://github.com/user-attachments/assets/6ba331c7-9060-4cd9-9c62-997c9baa836f" />

### Expected Behaviour

Ensure the avatar group accessible label is descriptive and meaningful.

### UI5 Web Components for React Version

2.7.1

### UI5 Web Components Version

2.7.2

### Browser

Chrome

### Operating System

Mac

### Relevant log output

```shell

```

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by Todor-ads** [comment]
Hello @SAP/ui5-webcomponents-topic-p  ,
could you please check this issue?
Thanks,
Todor

**Comment by yanaminkova** [comment]
Hello @summer2200,

The `ui5-avatar-group` component has been implemented according to accessibility specifications. It is designed to announce hidden items, even when there are none.

Thanks and Regards,
Yana
